src/GitReportCL/RunProcess.cs

```diff
@@ -0,0 +1,105 @@
+﻿
+
+namespace GitReportCL
+{
+    using System;
+    using System.Diagnostics;
+    using System.IO;
+
+    class RunProcess
```

Comment:
* Should be a `noun` class name should not have verbs `Run`
* Class name indicate that is used to run ANY process, but methods have `git` hard-coded, also only possible parameters are hardcoded to be git-specific.